x-pack/test/functional_search/tests/search_index_details.ts

```diff
@@ -38,28 +39,70 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
         });
 
         // Create a space with the search solution and navigate to its home page
-        ({ cleanUp, space: spaceCreated } = await spaces.create({
-          name: 'search-ftr',
+        ({ cleanUp: cleanUpSpace, space: spaceCreated } = await spaces.create({
+          name: 'solution-nav-search-index-details-ftr',
           solution: 'es',
         }));
+
         await pageObjects.searchApiKeys.deleteAPIKeys();
+        await es.indices.create({ index: indexName });
       });
 
       after(async () => {
         // Clean up space created
-        await cleanUp();
+        await cleanUpSpace();
         await esDeleteAllIndices(indexName);
       });
+      describe('index management index list page', () => {
```

Comment:
Maybe let's consider to move this tests in a separate file, because this file is related to search_index_details and tests about index management page?

Reply:
It also tests case when navigating to search_index details page from index management list page. But I don't have strong opinion on keeping it here. Will move.

Reply:
moved to separate file c605e2b13d57d2e498824257b7e0911c57586654

Reply:
moved to new file in 0e41d0a08deaf527e2622dd2ef9fce0fdc5d2b94

Reply:
Also moved index management tests in classic nav to new file 5e0e326c82c05d108622aed3fa6493ebb106f1e4